server/src/analyzer/commentUtils.ts

```diff
@@ -59,10 +59,20 @@ function _applyStrictSettings(settings: DiagnosticSettings) {
 }
 
 function _parsePyrightComment(commentValue: string, settings: DiagnosticSettings) {
-    // Is this a pyright-specific comment?
+    let prefixLength = 0;
+    // Is this a pyright or mspython-specific comment?
     const pyrightPrefix = 'pyright:';
```

Comment:
No need to change this if you don't want, but there's a bit cleaner way to do this.
```
const validPrefixes = ['pyright:', 'mspython:'];
const prefix = validPrefixes.find(p => commentValue.startsWith(p));
if (prefix) {
   ...
}
```

Reply:
Sure